### DISCUSSION TITLE
Is there a way  to generate thumbnails?

### DISCUSSION THREAD

**Discussion by jayanthymohit6** [RussCoder/djvujs#84] [Q&A]
const docDjvu = new DjVu.Document(xhr.response);

I did try to create a png but could'nt find the createPngObjectUrl method  .Is there any way we can generate pngs?

**Comment by RussCoder** [comment]
Yes, but it's a method of the DjVuPage class.
Here is the [docs](https://github.com/RussCoder/djvujs/blob/master/library/API.md). The link to this documentation can be found on the website too. 

For example: 

```js
const doc = new DjVu.Document(arrayBuffer);
const page = await doc.getPage(1);
const pngObject = await page.createPngObjectUrl();
```
